gstd/src/msg/async.rs

```diff
@@ -33,6 +34,23 @@ use core::{
 };
 use futures::future::FusedFuture;
 
+fn poll<F, R>(waiting_reply_to: MessageId, cx: &mut Context<'_>, f: F) -> Poll<Result<R>>
+where
+    F: Fn(Vec<u8>) -> Result<R>,
+{
+    match signals().poll(waiting_reply_to, cx) {
+        ReplyPoll::None => panic!("Somebody created CodecMessageFuture with the MessageId that never ended in static replies!"),
```

Comment:
Need to choose more neutral future name.
```suggestion
        ReplyPoll::None => panic!("Somebody created a future with the MessageId that never ended in static replies!"),
```